jpa/hibernate5-legacy/src/main/java/org/jboss/as/jpa/hibernate5/infinispan/InfinispanRegionFactory.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2013, Red Hat Middleware LLC, and individual contributors
```

Comment:
should be Copyright 2018.

Reply:
This was an existing file that was moved to a new module - so the original year is correct.